### ISSUE TITLE
Possible addition to warning text in `cylc validate`?

### ISSUE THREAD

**Issue by jonnyhtw** [cylc/cylc-rose#254]
hi there, 

### Problem

in many of my (climate modelling) workflows we have dates like this in our `rose-suite.conf` files...

```
[template variables]
EXPT_BASIS=1978,09,01,00,00
```

when running `cylc validate` on a workflow containing this, a warning is thrown...

```
> cylc val .
WARNING - Support for integers with leading zeros was dropped in Jinja2 v3. Rose will extend support until a future version.
    Please amend your Rose configuration files e.g:
     * 00 => 0
     * 09 => 9
     * 01 => 1
```

1. firstly, it wasn't clear to me that the examples in the warning message were the *actual* errors in the workflow definition and...
2.  secondly i'd suggest making it clearer to users that this warning also applies to lists of integers.

### Proposed Solution

i suggest making a couple of very small changes to the warning message so for this particular example something like...

```
WARNING - Support for integers with leading zeros (including lists of integers) was dropped in Jinja2 v3. Rose will extend support until a future version.
    Please amend your Rose configuration files, which currently contain:
     * 00 => 0
     * 09 => 9
     * 01 => 1
```

cheers, 

jonny

**Comment by hjoliver** [comment]
Rose devs: I think @jonnyhtw's suggestion is reasonable, perhaps with slightly tweaked wording: while it should be obvious that a warning about Jinja2 integer format also applies to lists (or any other collection) of Jinja2 integers, it's not exactly obvious that a comma-separated list in the `rose-suite.conf` config file gets parsed into a Jinja2 list of integers, as opposed a Jinja2 string that's left for user Jinja2 code to handle as they like.

**Comment by oliver-sanders** [comment]
Closes #255
